data-prepper-api/src/main/java/org/opensearch/dataprepper/model/peerforwarder/RequiresPeerForwarding.java

```diff
@@ -30,4 +30,15 @@ public interface RequiresPeerForwarding {
     default boolean isApplicableEventForPeerForwarding(Event event) {
         return true;
     }
+
+    /**
+     * Determines if an event should be handled on the local node only
+     *
+     * @param event input event
+     *
+     * @return true if the event should be processed locally only
+     */
+    default boolean isForLocalProcessingOnly(Event event) {
```

Comment:
Should we use the existing `isApplicableEventForPeerForwarding`?

Put another way, do we desire to support isApplicableEventForPeerForwarding=false and isForLocalProcessingOnly=false? This would always skip.

Reply:
Yes. If both are false, it would skip